src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -91,6 +91,12 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
         [InlineData(SslProtocols.Tls12, true)]
         public async Task GetAsync_AllowedSSLVersion_Succeeds(SslProtocols acceptedProtocol, bool requestOnlyThisProtocol)
         {
+            if (ManagedHandlerTestHelpers.IsEnabled)
+            {
+                // TODO #21452: The managed handler is failing.
+                return;
+            }
```

Comment:
It might even be related to 7812 which is really an SCHANNEL bug with DHE ciphers